PocketKit/Sources/PocketKit/Settings/AccountViewModel.swift

```diff
@@ -15,4 +16,8 @@ class AccountViewModel {
     func signOut() {
         appSession.currentSession = nil
     }
+    
+    @Published var helpPresented = false
+    @Published var termsPresented = false
+    @Published var privacyPresented = false
```

Comment:
Just a style nit:

We generally follow the pattern of having boolean properties begin with `is` or `has`. e.g:

```swift
@Published var isPresentingHelp = false
```

Not strictly necessary, but this naming convention comes from Apple and most style guides and adds a bit more clarity to the purpose of the property.